rebar.config

```diff
@@ -37,10 +37,11 @@
  [{test,
    [{plugins, [{coveralls, {git, "https://github.com/emqx/coveralls-erl", {branch, "github"}}}]},
     {deps, [{meck, "0.8.13"},
-            {proper, "1.3.0"},
-            {conver, {git, "https://github.com/k32/conver-old", {branch, "master"}}}
```

Comment:
I found some rather buggy code in this library, and it doesn't verify the full spectrum of ops that mnesia supports, so it doesn't make sense to integrate it.